test/lib.spec.js

```diff
@@ -1073,6 +1073,7 @@ describe("encode <-> decode", () => {
       s: "hello",
     })
 
+    // @ts-ignore we are forcing a string to test decode throws
```

Comment:
```suggestion
    // @ts-expect-error we are forcing a string to test decode throws
```

If it's expected to fail we should use expect-error instead